megamek/src/megamek/common/EjectedCrew.java

```diff
@@ -17,10 +17,14 @@
 public class EjectedCrew extends Infantry {
     protected int originalRideId;
     protected String originalRideExternalId;
+    // set up movement for Aero pilots and vessel crews
+    protected int currentVelocity = 0;
+    protected int nextVelocity = currentVelocity;
```

Comment:
I can see where currentVelocity is used, but where is nextVelocity used?

Reply:
It isn't at present.  I was thinking ahead to ejected spacecraft crews when I put these in.  currentVelocity and nextVelocity work with non-vector aerospace movement, which I never use.  
I tested that enough to be satisfied that it doesn't break.  I'll remove this if I still haven't used it by the time I get escape pods working.